### ISSUE TITLE
Content disappears after saving

### ISSUE THREAD

**Issue by jan-dh** [verbb/vizy#28]
**Description**

I made a plain text block to output some twig code in a blog post and style it with prism.js  The content of the block is something like this:

```<table class="responsive-table">
{% for row in entry.tableField %}
  {% if loop.index == 1%}
  <thead>
    <tr>
      <th scope="col">{{row.col1}}</th>
      {% for i in 2..(tableSize) %}
        {% set colName = "col" ~ i %}
        {% set colValue = attribute(row, colName) %}
        {% set labelName = 'label' ~ i %}
        {% set emptyObject = emptyObject|merge({(labelName): (colValue)}) %}
        <th scope="col">{{colValue}}</th>
      {% endfor %}
    </tr>
  </thead>
  <tbody>
  {% else %} 
```

After saving the block and reloading the entry the content is gone:

![image](https://user-images.githubusercontent.com/5801781/119325057-c745ac80-bc80-11eb-9139-25b16e500dec.png)

![image](https://user-images.githubusercontent.com/5801781/119325095-d3ca0500-bc80-11eb-91a6-8d5685553d27.png)

I also get two errors from a vue file:

`ReferenceError: attribute is not defined` and `vue.js?v=1621609911:6 ReferenceError: row is not defined`

My guess would be that the unclosed `{% else %}` tag creates some issues when saving this field. I'm assuming it does get parsed somewhere before save, even though it's just a plain text field  

**Steps to reproduce**
1. Create a `code` block with a plain text field
2. Copy the code snippet I posted above and save the entry
3. Reload the page

**Additional info**
- Plugin version: dev-craft-3 as 1.0.1
- Craft version: 3.6.15
- Multi-site: Yes

**Comment by engram-design** [comment]
Ah, good point here, because we're using Vue, it's trying to parse that as Vue template code - not what we want at all.

Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.2",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.0.3](https://github.com/verbb/vizy/releases/tag/1.0.3)
